src/azure-cli/azure/cli/command_modules/eventhubs/commands.py

```diff
@@ -177,3 +185,15 @@ def load_command_table(self, _):
         g.command('list', 'list_by_namespace')
         g.show_command('show', 'get')
         g.command('delete', 'delete')
+
+# ApplicationGroup Region
+    with self.command_group('eventhubs namespace application-group', eh_application_group_util, resource_type=ResourceType.MGMT_EVENTHUB, client_factory=application_group_mgmt_client_factory) as g:
+        g.custom_command('create', 'cli_appgroup_create')
+        g.custom_command('update', 'cli_appgroup_update')
+        g.show_command('show', 'get')
+        g.command('delete', 'delete')
+        g.command('list', 'list_by_namespace')
+
+    with self.command_group('eventhubs namespace application-group application-group-policy', eh_application_group_util, resource_type=ResourceType.MGMT_EVENTHUB, client_factory=application_group_mgmt_client_factory) as g:
```

Comment:
I would suggest to use `eventhubs namespace application-group policy` as command group name. What do you think?

Reply:
yes, that would be easier to use. I will make the changes and push it. Thanks !